src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -110,6 +113,14 @@ def collect_data(geop_result, geojson):
     z.StreamLength = stream_length(geom)   # Meters
     z.n42b = round(z.StreamLength / 1000)  # Kilometers
 
+    # Values derived from (NLCD, Slope, Kfactor) datasets
```

Comment:
Consider adding the values that depend on `geop_result` after National Weather Dataset (line 134 in this file), to keep all the geoprocessing dependent code _after_ the independent code.